### ISSUE TITLE
NTR: ZAP-70 protein expression decreased by immunocytochemistry testing of CD4 T-cells (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8725]
New term request ZAP-70 protein expression decreased by immunocytochemistry testing of CD4 T-cells (elthat_100612015049):

**Comment by pnrobinson** [comment]
@MickeySegal
this is from https://omim.org/entry/269840 I take it?
This sounds like a one-off research only investation or is it offered as a clinical test? I could not find anything.

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK20221/
ZAP-70 protein expression. Testing of CD4+ T cells reveals absence or near absence of ZAP-70 protein in most affected individuals.

**Comment by pnrobinson** [comment]
I still think this was not a clinical test. Two of the aticle cited above (Chan 2016, and Gavine 2017) do not present the ZAP70 as a test in any of the tables and I cannot find a CD4 test by scanning the text. I think this is only something one would do if one was writing a paper about this disease, but otherwise people would do a genetic sequencing and stop there. Some of the tests reported in GeneReviews do seem to be mainly research based, I am not sure that all authors make this distinction?

**Comment by MickeySegal** [comment]
Quest does a ZAP-70 test: https://testdirectory.questdiagnostics.com/test/test-detail/16000/zap-70?cc=MASTER

**Comment by pnrobinson** [comment]
I stand corrected!

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001096
- label: Decreased T-cell ZAP-70 protein expression
- parents: [Abnormal T cell morphology (HP:0002843)](https://hpo.jax.org/app/browse/term/HP:0002843)
- pmid: PMID:20301777
- def: Reduction in the level of ZAP-70 protein CD4-positive T cells.
- comment: Testing of CD4+ T cells reveals absence or near absence of ZAP-70 protein in most individuals affected by ZAP70-related combined immunodeficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8725
- synonyms:
